release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2573,39 +2573,49 @@ In the following tables, features are marked with the following statuses:
 |====
 
 [discrete]
-=== Hosted control planes Technology Preview features
+=== {hcp-capital} Technology Preview features
 
-.Hosted control planes Technology Preview tracker
+.{hcp-capital} Technology Preview tracker
 [cols="4,1,1,1",options="header"]
 |====
 |Feature |4.15 |4.16 |4.17
 
-|Hosted control planes for {product-title} on {aws-first}
-|Technology Preview
-|Technology Preview
+|{hcp-capital} for {product-title} on {aws-first}
 |Technology Preview
+|General Availability
+|General Availability
+
+|{hcp-capital} for {product-title} on bare metal
+|General Availability
+|General Availability
+|General Availability
 
-|Hosted control planes for {product-title} using non-bare metal agent machines
+|{hcp-capital} for {product-title} on {VirtProductName}
+|General Availability
+|General Availability
+|General Availability
+
+|{hcp-capital} for {product-title} using non-bare metal agent machines
```

Comment:
```suggestion
|{hcp-capital} for {product-title} using non-bare-metal agent machines
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words
See SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_b

Reply:
I see. `non-bare-metal` does sound fair to use. I am going to have to leave it as is here for the similarity/consistency though. Because all of the other instances are non-bare metal. I will try to see if this can be taken in a follow up PR.